segmentation/utils/data_utils.py

```diff
@@ -7,14 +7,60 @@
 import numpy as np
 import xarray as xr
 import scipy.ndimage as nd
+from mibidata import tiff
 
 import skimage.filters.rank as rank
 import scipy.ndimage as nd
 
 
+def load_imgs_from_mibitiff(mibitiff_paths, channels=None, load_axis="fovs"):
+    """Load images from a series of MIBItiff files.
+
+    This function takes a set of MIBItiff files and load the images into an
+    xarray. The type used to store the images will be the same as that of the
+    MIBIimages stored in the MIBItiff files.
+
+    Args:
+        mibitiff_paths: list of MIBItiff files to load.
+        channels: optional list of channels to load. Defaults to `None`, in
+            which case, all channels in the first MIBItiff are used.
+        load_axis: axis that images will get loaded onto. Must be one of
+            ["fovs", "stacks"].
+
+    Returns:
+        img_xr: xarray with shape [fovs, tifs, x_dim, y_dim]
+    """
+
+    # sanity check
+    if not np.isin(load_axis, ["fovs", "stacks"]):
+        raise ValueError("Invalid value for load_axis, must be one of [fovs, stacks]")
+
+    # if no channels specified, get them from first MIBItiff file
+    if channels is None:
+        channels = img_data[0].channels
```

Comment:
I don't think img_data has been defined yet, right?

Reply:
You're right. I fixed this and added a test for this case.